client/src/modules/Students/components/StudentsTable/renderers.tsx

```diff
@@ -0,0 +1,95 @@
+import { Space, Tag, Tooltip } from 'antd';
+import { ColumnsType } from 'antd/lib/table';
+import { CourseDto, UserStudentCourseDto, UserStudentDto } from 'api';
+import { GithubUserLink } from 'components/GithubUserLink';
+import { getColumnSearchProps } from 'components/Table';
+
+export enum ColumnKey {
+  student = 'student',
+  onGoingCourses = 'onGoingCourses',
+  previousCourses = 'previousCourses',
+  country = 'country',
+  city = 'city',
+}
```

Comment:
Let's use more widely used enum style: it's members are usually in camel case or upper case  
```suggestion
export enum ColumnKey {
  Student = 'student',
  OngoingCourses = 'ongoingCourses',
  PreviousCourses = 'previousCourses',
  Country = 'country',
  City = 'city',
}
```